pkg/service/service_config/handler.go

```diff
@@ -52,11 +52,30 @@ func (p *Server) SetServiceConfig(ctx context.Context, req *pb.SetServiceConfigR
 		if err != nil {
 			return nil, gerr.NewWithDetail(ctx, gerr.Internal, err, gerr.ErrorSetNotificationConfig)
 		}
-		response, err := nfClient.SetServiceConfig(ctx, OpToNfConfig(req.NotificationConfig))
+		reqServiceCfg := &nfpb.ServiceConfig{
```

Comment:
Why not use OpToNfConfig(req.NotificationConfig)?

Reply:
![Screenshot from 2019-03-27 17-10-07](https://user-images.githubusercontent.com/5917832/55063608-4a741380-50b3-11e9-85af-a6dcb7f7b145.png)

fix it.